README.rst

```diff
@@ -112,12 +112,62 @@ You can use the library to send and receive requests and responses like so,
 
         return blog.dumps(response)
 
+Note that this example sends and receives just the request/response payload. It
+does not wrap the payload in a message envelope as expected by Apache Thrift.
+If you want to send or receive standard Apache Thrift requests to talk to other
+Apache Thrift services, you have to use the ``loads.message`` and
+``dumps.message`` APIs. For example,
+
+.. code-block:: python
+
+    # client.py
+
+    def new_post():
+        post = blog.PostDetails(...)
+        request = BlogService.newPost.request(post)
+        payload = blog.dumps.message(request)
```

Comment:
It `message` HTTP-specific? Would `blog.dumps.http` (or `blog.http.dumps`) be more obvious?